### ISSUE TITLE
Buttons are too large in when maximizing window

### ISSUE THREAD

**Issue by Huite** [rchartra/NcCut#36]
Related to: https://github.com/openjournals/joss-reviews/issues/7185

Everything seems in proportion when initially opening:

![image](https://github.com/user-attachments/assets/4a927610-33ac-4726-b0c1-6a9fa3d60633)

But when I maximize the window (on my 1920 x 1200 monitor):

![image](https://github.com/user-attachments/assets/6510d0ac-19a4-48ad-bac5-22ca5e072848)

Far too much screen estate is taken up by the buttons. The font is also blown up disproportionately, compare with the comparatively tiny file path (which has reasonable font size btw).

This feels especially unsatisfactory in the Plot Menu:

![image](https://github.com/user-attachments/assets/c49ed040-0540-4867-8b9a-d46464cac9cf)

The plots are getting squashed because "Select Transects" is so wide.

This also relates to #33, where the variable names font is much smaller.

My experiences with PyQt is that setting up size hinting can be quite fiddly and tedious. For NcCut, I think the most important thing is that your plotting panes get the maximum screen estate possible. To this end, I think you could also make a pragmatic decision and just set a static size on all your buttons and other UI elements. Secondly, I would recommend you choose one font size and make it consistent throughout the application.

Potentially, to allow a bit more flexibility for colleagues with poor eye-sight: make the font size configurable, and make the button size dependent on the font size.

**Comment by rchartra** [comment]
Good point, I agree that in both the main screen and the plotting popup menu as much space as possible should be given to the plot. Kivy is also kind of finicky with it's size hinting but once you figure out the pattern it's not too bad. So I'll figure out a way to make the layout better. I also like the idea of making the font size configurable. I'll add it as an option for the nccut_config.toml file.

**Comment by rchartra** [comment]
Resolved in v0.4.7
